src/cmd/flux-jobs.py

```diff
@@ -256,10 +256,16 @@ def parse_args():
     )
     parser.add_argument(
         "-n",
-        "--suppress-header",
+        "--no-header",
         action="store_true",
         help="Suppress printing of header line",
     )
+    # --suppress-header, older legacy name
+    parser.add_argument(
+        "--suppress-header",
+        action="store_true",
```

Comment:
you can use the [dest](https://docs.python.org/3/library/argparse.html#dest) option to `add_argument` to store true in `no_header` instead of `suppress_header` so that this just becomes a hidden alternate for `--no-header`, e.g.

```python
parser.add_argument(
    "--suppress-header",
    action="store_true",
    dest="no_header",
    help=argparse.SUPPRESS
)
```

Then remove references to `args.suppress_header` everwhere.